subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,28 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+The strategy of running builds in single use environments in a CI/CD setup is a challenge for build tools as a lot of state that could be reused between builds has to be rebuilt for each execution.
```

Comment:
I think the narrative could go like this:
- It's increasingly common to use a fresh environment for each build on CI (ephemeral builds)
- While it can help to increase build reliability, it can also significant overhead to each CI build making it longer
- One of the most important overhead aspects to keep in mind is dependency downloads. It's very inefficient for each CI build to download all project dependencies 
- In order to address this problem, the dependency cache can be relocated to another directory or host since Gradle 6.1
- ... explain what exactly you can use it to address the problem (e.g. move pre-populated cache to a Docker image that you use for your ephemeral builds)